src/nvim/main.c

```diff
@@ -1624,9 +1619,8 @@ static void open_script_files(mparm_T *parmp)
   if (parmp->scriptout) {
     scriptout = os_fopen(parmp->scriptout, parmp->scriptout_append ? APPENDBIN : WRITEBIN);
     if (scriptout == NULL) {
-      os_errmsg(_("Cannot open for script output: \""));
-      os_errmsg(parmp->scriptout);
-      os_errmsg("\"\n");
+      fprintf(stderr, _("Cannot open for script output: \""));
```

Comment:
This message is translated, so I think it's better to make it a format argument.
```suggestion
      fprintf(stderr, "%s", _("Cannot open for script output: \""));
```